x-pack/plugins/observability/public/plugin.ts

```diff
@@ -146,9 +145,10 @@ export interface ObservabilityPublicPluginsStart {
   aiops: AiopsPluginStart;
   serverless?: ServerlessPluginStart;
 }
-
 export type ObservabilityPublicStart = ReturnType<Plugin['start']>;
 
+const LOG_EXPLORER_LOCATOR_ID = 'LOG_EXPLORER_LOCATOR';
```

Comment:
**nit:** this can be imported from `@kbn/deeplinks-observability`, which is the source of truth used to register the locator.